src/main/java/spotify/utils/HttpUtil.java

```diff
@@ -9,8 +9,24 @@
 
 import java.io.IOException;
 
+/**
+ * This class contains utility functions for use in the Authorization Code Flow.
```

Comment:
It is currently only used in `AuthorizationCodeFlow` but it isn't necessary only used there.